### DISCUSSION TITLE
Remote control and technic hub

### DISCUSSION THREAD

**Discussion by OlafZarthan** [pybricks/support#631] [Q&A]
Hello

Im a totally newbie on using that coding technology (so complicated for me) for controling my technic MOC, and i have a most probably stupid question.

I need to control the 4 port on the technic hub for 4 differents engine, 4 different fonctions in same time (run, steering, lift arm, excavator) and then use 2 remote control (88010) is it possible to do that? and how? 

Sorry for stupid question and for my bad english (im french speacker)

**Comment by dlech** [comment]
Using two remote controls with one hub is not currently possible. But it is possible write a program, for example, that could change which motor is being controlled when you press a button.

For controlling multiple motors at the same time, there is the `Motor.run()` method and other methods with the `wait=False` parameter.

You could start with an example program like https://pybricks.com/projects/sets/technic/42129-mercedes-benz-zetros/powered-up-remote/.

**Comment by ZPhilo** [comment]
Another example of Pybricks code to control a complex device (42114) from a single remote: https://github.com/repkovsky/technic/tree/main/42114-volvo-articulated-hauler
...and if needed, je parle français ;)

**Reply by OlafZarthan** [reply]
> Another example of Pybricks code to control a complex device (42114) from a single remote: https://github.com/repkovsky/technic/tree/main/42114-volvo-articulated-hauler ...and if needed, je parle français ;)

Oh oui ça m'aiderait si je peux passer en français, je galère assez comme ça mdr

**Reply by OlafZarthan** [reply]
> Another example of Pybricks code to control a complex device (42114) from a single remote: https://github.com/repkovsky/technic/tree/main/42114-volvo-articulated-hauler ...and if needed, je parle français ;)

Donc il serait techniquement possible avec une seule manette de controler par exemple propulsion et direction (en port A et B). Et en poussant sur le bouton vert octroyer bras de levage et godet (en port C et D) et pouvoir switcher comme ça?

**Reply by ZPhilo** [reply]
Oui, exactement. C'est ce que permet le code que j'ai cité. J'avais fait avant de découvrir ça une version beaucoup plus rustique visible ici: 
https://www.youtube.com/watch?v=igGqzSjl0U0
A l'époque il n'était pas possible de controler la LED de la télécommande, mais on voit bien l'utilisation du bouton vert pour passer du mode conduite au mode lame.

**Comment by OlafZarthan** [comment]
D'accord, c'est un bon début pour la solution à mon problème. ahahha. Mais il faut que j'apprenne à coder parce qu'actuellement j'utilise les codes tout fait qui sont dispo. Tu connais un genre de tutoriel ou de site qui montre les bases? Merci
